src/openlcb/MemoryConfig.hxx

```diff
@@ -511,25 +511,28 @@ public:
         size_t len = message->data()->payload.size();
         const uint8_t *bytes = (const uint8_t *)message->data()->payload.data();
         uint8_t cmd = ((len >= 2) && (client_ != nullptr)) ? bytes[1] : 0;
+        bool is_client_command = false;
+        if ((cmd < 0x80) && (cmd & 0x10))
```

Comment:
Add enum values for the top and bottom of the client range directly into MemoryConfigDefs for two reasons:
1. So the values here are not magic numbers
2. To help ensure that anyone who might change MemoryConfigDefs knows about the grouping for client commands

Reply:
done